cmd/guaccsub/cmd/root.go

```diff
@@ -48,6 +48,7 @@ var rootCmd = &cobra.Command{
 			logger.Fatalf("unable to create csub server: %v", err)
 		}
 
+		// csubServer.ctx = ctx
```

Comment:
in that case, this will not be needed.

Reply:
Understood. I see how your fix will prevent needing a change to guacsub command.